builtin_interfaces/Time stamp
float32 puck_x
float32 puck_y
float32 puck_vx
float32 puck_vy

float32[] agent_x
float32[] agent_y
float32[] agent_vx
float32[] agent_vy